docs/sphinx/source/changelog/pending.rst

```diff
@@ -2,6 +2,11 @@
 Pending
 *******
 
+Enhancements
+------------
+* Add new function :py:func:`~rdtools.filtering.geometric_clip_filter` (:pull:`200`).
```

Comment:
@kanderso-nrel I did end up updating the soiling script because it was failing pep8 protocol. I can pull back a commit for that. Thanks for having a look at this--I'll look at what I can do to clean up code redundancies + unit testing this afternoon.